client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/composables/BpDropdownMenuItem.kt

```diff
@@ -25,8 +25,8 @@ fun BpDropdownMenuItem(
 ) {
     Column(modifier = modifier) {
         DropdownMenuItem(
+            modifier= Modifier.padding(horizontal = LocalDimensions.current.space8),
```

Comment:
```suggestion
            modifier= Modifier.padding(horizontal = 8.kms),
```
don't use spacings only use hard coded values in kms measurment

Reply:
after Merage i will change them